From 1095f1025c5ecdb7f0438e75e8cb5e735d226a28 Mon Sep 17 00:00:00 2001
From: Igor Opaniuk <igor.opaniuk@foundries.io>
Date: Tue, 15 Feb 2022 21:26:23 +0200
Subject: [PATCH 312/322] [FIO internal] imx8qm-mek: select FSL_CAAM only if
 HAS_CAAM is y

Select FSL_CAAM and FSL_BLOB only when HAS_CAAM is set.

Signed-off-by: Igor Opaniuk <igor.opaniuk@foundries.io>
---
 arch/arm/mach-imx/imx8/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-imx/imx8/Kconfig b/arch/arm/mach-imx/imx8/Kconfig
index 695231af779..0d590241bd0 100644
--- a/arch/arm/mach-imx/imx8/Kconfig
+++ b/arch/arm/mach-imx/imx8/Kconfig
@@ -97,8 +97,8 @@ config TARGET_IMX8QM_MEK
 	bool "Support i.MX8QM MEK board"
 	select BOARD_LATE_INIT
 	select IMX8QM
-	imply FSL_CAAM
-	imply FSL_BLOB
+	imply FSL_CAAM if HAS_CAAM
+	imply FSL_BLOB if HAS_CAAM
 	select ARCH_MISC_INIT
 	select SPL_CRYPTO_SUPPORT if SPL
 
-- 
2.34.1

